Assignment4/Assignment4.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+
+function checkError(error){
+  if(error)
+    throw error;
+}
+
+function print(string)
+{
```

Comment:
Block to be started in same line as function declaration.
Problem Type: Linting